src/Agent.Worker/Build/SvnSourceProvider.cs

```diff
@@ -1,78 +1,90 @@
+using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.TeamFoundation.Build.WebApi;
+using Microsoft.TeamFoundation.DistributedTask.WebApi;
 using Microsoft.VisualStudio.Services.Agent.Util;
 using Newtonsoft.Json;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
 
 namespace Microsoft.VisualStudio.Services.Agent.Worker.Build
 {
     public sealed class SvnSourceProvider : SourceProvider, ISourceProvider
     {
         public override string RepositoryType => WellKnownRepositoryTypes.Svn;
 
-        public async Task GetSourceAsync(IExecutionContext executionContext, ServiceEndpoint endpoint, CancellationToken cancellationToken)
+        public async Task GetSourceAsync(
+            IExecutionContext executionContext, 
+            ServiceEndpoint endpoint, 
+            CancellationToken cancellationToken)
         {
-            Trace.Verbose("Entering SvnSourceProvider.GetSourceAsync");
+            Trace.Entering();
 
-            // Validate args.
+// Validate args.
             ArgUtil.NotNull(executionContext, nameof(executionContext));
             ArgUtil.NotNull(endpoint, nameof(endpoint));
 
             ISvnCommandManager svn = HostContext.CreateService<ISvnCommandManager>();
             svn.Init(executionContext, endpoint, cancellationToken);
 
             // Determine the sources directory.
-            string sourcesDirectory = executionContext.Variables.Build_SourcesDirectory;
+            string sourcesDirectory = GetEndpointData(endpoint, Constants.EndpointData.SourcesDirectory);
             executionContext.Debug($"sourcesDirectory={sourcesDirectory}");
             ArgUtil.NotNullOrEmpty(sourcesDirectory, nameof(sourcesDirectory));
 
-            string sourceBranch = executionContext.Variables.Build_SourceBranch;
+            string sourceBranch = GetEndpointData(endpoint, Constants.EndpointData.SourceBranch);
             executionContext.Debug($"sourceBranch={sourceBranch}");
 
-            string revision = executionContext.Variables.Build_SourceVersion;
+            string revision = GetEndpointData(endpoint, Constants.EndpointData.SourceVersion);
             if (string.IsNullOrWhiteSpace(revision))
             {
                 revision = "HEAD";
             }
+
             executionContext.Debug($"revision={revision}");
 
             bool clean = endpoint.Data.ContainsKey(WellKnownEndpointData.Clean) &&
-                StringUtil.ConvertToBoolean(endpoint.Data[WellKnownEndpointData.Clean], defaultValue: false);
+                         StringUtil.ConvertToBoolean(endpoint.Data[WellKnownEndpointData.Clean], defaultValue: false);
             executionContext.Debug($"clean={clean}");
 
             // Get the definition mappings.
-            List<SvnMappingDetails> allMappings = JsonConvert.DeserializeObject<SvnWorkspace>(endpoint.Data[WellKnownEndpointData.SvnWorkspaceMapping]).Mappings;
+            List<SvnMappingDetails> allMappings =
```

Comment:
I agree with you. The resharper on my IDE is the culprit here. My stylecop settings are busted here. Will fix this.
Thanks for pointing it out.